keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/CombinedRepository.kt

```diff
@@ -258,8 +260,11 @@ class CombinedRepository(
   override fun deleteResource(id: String) =
     resourceRepository.delete(id)
 
-  override fun deleteResourcesByApplication(application: String): Int =
-    resourceRepository.deleteByApplication(application)
+  override fun applicationEventHistory(application: String, limit: Int): List<KeelApplicationEvent> =
+    resourceRepository.applicationEventHistory(application, limit)
+
+  override fun applicationEventHistory(application: String, downTo: Instant): List<KeelApplicationEvent> =
```

Comment:
what does `downTo` mean?

Reply:
"Starting at the most recent event, **down to** this earlier timestamp, give me the application event history".

Reply:
where is this comment from?

Reply:
Sorry, I'm not following... Do you mean you'd like this added as a function comment somewhere? I can add that. Otherwise, do you have a suggestion for a better name for `downTo`, perhaps?